### ISSUE TITLE
Sharing: Create sharing link for multiple albums at once

### ISSUE THREAD

**Issue by ScuttleSE** [photoprism/photoprism#1162]
I can share one picture easily.
I can share multiple pictures with the same theme by easily collecting them in one Album and then share that album

What I lack is the ability to collect multiple Albums in one... Gallery? and share that.

I have recently scanned thousands of old negatives with family pictures. They are, on disk, separated in different folders. Sharing them with my family is next to impossible. Right now I can either share one link with album with 10.000+ pictures in it, or share 200+ links with them. Neither is really practical.

So... It would be nice with a way to collect albums in a Gallery or something, and then have the ability to share it the same way as a Album today.

**Comment by graciousgrey** [comment]
I don't think we need something like a collection of albums I rather see the following possible solutions:
1) Use [Multi User Management](https://github.com/photoprism/photoprism/issues/98): Create a "family user" --> Set the rights for this user --> Decide which photos can be viewed by this user --> Share the credentials of this user with your family
2) Ability to select multiple albums and create one sharing link for all of those

What do you think?

**Comment by FreshMosh** [comment]
I'd like the 2nd option better as it would not be limited to "trusted users" who need to have access to the shared user prior but allows for just sharing a bunch of albums occasionally even to more or less strangers. 

This is just an idea for the long term target with this feature:
It would also be nice if those multiple albums would be displayed as one single album, preferably with the (optional) ability to show the respective "sub"-album name as a subheading (which would allow to better know the context of the pictures, which might or might not be important)

Like:
# Name of the first album
[Pic] [Pic] [Pic] 
[Pic] [Pic] 
# Name of the 2nd album
[Pic] [Pic] [Pic] 
# Name of the 3rd album
[Pic]

**Comment by ScuttleSE** [comment]
> I don't think we need something like a collection of albums I rather see the following possible solutions:
> 
> 1. Use [Multi User Management](https://github.com/photoprism/photoprism/issues/98): Create a "family user" --> Set the rights for this user --> Decide which photos can be viewed by this user --> Share the credentials of this user with your family
> 2. Ability to select multiple albums and create one sharing link for all of those
> 
> What do you think?

I would say that the possibility to share multiple albums the same way a single album is share:able today is a much easier way to share pictures with family and friends.

Adding the complexity-layer of creating users, assigning permissions and selecting which folders that user is able to see is usable too I guess, but right now one of the draws of Photoprism is how easy it is to create a shared album.

**Comment by graciousgrey** [comment]
I agree, the second suggestion is much more convenient. 
I outlined the first one as well, as it might be that we implement user management before we find time for this ticket

**Comment by rom4nik** [comment]
I've found out that as a workaround you can share albums separately with common secret, and then use `https://domain.tld/s/secret/` (remove album part from the generated URL) to see those albums in one place.
